clients/hadoopfs/src/main/java/io/lakefs/LakeFSFileStatus.java

```diff
@@ -31,4 +28,59 @@ public String getChecksum() {
     public String getPhysicalAddress() {
         return physicalAddress;
     }
+
+    public static class LakeFSFileStatusBuilder {
```

Comment:
I've also seen this style:
```suggestion
    public static class Builder {
```
to avoid having the repeat "LakeFSFileStatus" (it's there in the outer classname).